src/main.ts

```diff
@@ -65,6 +66,27 @@ export function portMap(channels: MessageChannels, portNumber: 1 | 2): MessagePo
 }
 
 export function makeSandboxDriver(): Driver<undefined, Sources> {
+
+
+  let raf: boolean;
+  
+  const raf$ = xs.create({
+    start(listener) {
+      raf = true;
+      const r = () => requestAnimationFrame((e) => {
```

Comment:
fair point, I just imagined that one to many drivers may be interested in the raf signal. perhaps we could simulate it for non browser contexts?